guides/manual-installation.md

```diff
@@ -70,6 +70,28 @@ Problem: nothing provides 'group(prometheus)' needed by the to be installed gola
  Solution 2: break golang-github-prometheus-prometheus-2.37.6-150100.4.17.1.x86_64 by ignoring some of its dependencies
 ```
 
+Change prometheus configuration by replacing the configuration at `/etc/prometheus/prometheus.yml` with:
+
+```bash
+global:
+  scrape_interval: 30s
+  evaluation_interval: 10s
+
+scrape_configs:
+  - job_name: "http_sd_hosts"
+    honor_timestamps: true
+    scrape_interval: 30s
+    scrape_timeout: 30s
+    scheme: http
+    follow_redirects: true
+    http_sd_configs:
```

Comment:
This `http_sd_config` (and I think the `follow_redirects` too) is apparently not supported in the prometheus version we can get in SP3

Reply:
@rtorrero Let's keep thas as is while we submit backport prometheus until SP3